src/components/cc-env-var-form/cc-env-var-form.js

```diff
@@ -65,41 +59,27 @@ export class CcEnvVarForm extends LitElement {
     /** @type {string} Defines application name used in some heading/description (depending on context). */
     this.appName = '?';
 
-    /** @type {ContextType} Defines where the form will be used so it can display the appropriate heading and description. */
+    /** @type {EnvVarFormContextType} Defines where the form will be used, so it can display the appropriate heading and description. */
     this.context = null;
 
-    /** @type {boolean} Whether to displays a loading error message. */
-    this.error = false;
-
     /** @type {string|null} Sets a text to be used as a header title. */
     this.heading = null;
 
-    /** @type {ParserOptions} Sets the options for the variables parser. */
-    this.parserOptions = { mode: null };
-
     /** @type {boolean} Sets `readonly` attribute input and hides buttons. */
     this.readonly = false;
 
     /** @type {boolean} Displays the restart app button. */
     this.restartApp = false;
 
-    /** @type {boolean} Enables saving state (form is disabled and loader is displayed). */
-    this.saving = false;
+    /** @type {EnvVarFormState} Sets variables form state. */
+    this.state = { type: 'loading' };
 
-    // this.variables is let to null by default (this triggers skeleton screen)
-    /** @type {Variable[]|null} Sets the list of variables. */
-    this.variables = null;
+    this._editorsState = { type: 'loading' };
 
     /** @type {Variable[]|null} */
     this._currentVariables = null;
```

Comment:
question: it's no longer in the reactive properties, is it normal?

Reply:
It doesn't need to be reactive. The reactivity is done with the new `_editorsState` property.